lua/entities/gmod_wire_expression2/core/find.lua

```diff
@@ -442,6 +442,29 @@ e2function entity findPlayerByName(string name)
 	return findPlayer(name)
 end
 
+--- Returns the player with the given SteamID
+e2function entity findPlayerBySteamID(string id)
+	if query_blocked(self, 1) then return NULL end
+	return player.GetBySteamID(id) or NULL
+end
+
+--- Returns the player with the given SteamID64
+e2function entity findPlayerBySteamID64(string id)
+	if query_blocked(self, 1) then return NULL end
+	return player.GetBySteamID64(id) or NULL
+end
+
+--- Returns the player with the given UserID
+e2function entity findPlayerByUserID(number id)
+	if query_blocked(self, 1) then return NULL end
+	for k, v in ipairs(player.GetAll()) do
```

Comment:
> P.S. I remember seeing "Lua JIT is disabled." message in Console, so...

LuaJIT is only disabled if the server owner disables it. 


> Have you done the benchmark, ever, in GMod between those iterators?

Yes, I have. ipairs is always faster in my testing. When testing without LuaJIT outside of Gmod, ipairs seems to be faster than pairs on smaller tables (<15k values). I don't have time to do the tests again, although I probably will tomorrow if you'd like.